docs/talks/abstract-nesi-august-2017.md

```diff
@@ -0,0 +1,28 @@
+# Introducing the Stencila project
```

Comment:
Could you leave this file out of the PR - don't think it's really relevant to this particular repo